aiplatform/api/AIPlatform.Samples/PredictChatPromptSample.cs

```diff
@@ -0,0 +1,151 @@
+
+        // Or, you can construct Protobuf directly.
+        var instance = new wkt::Value
+        {
+            StructValue = new()
+            {
+                Fields =
+                {
+                    ["context"] = wkt::Value.ForString("My name is Miles. You are an astronomer, knowledgeable about the solar system."),
+                    ["examples"] = wkt::Value.ForList(wkt::Value.ForStruct(new()
+                    {
+                        Fields =
+                        {
+                            ["input"] = wkt::Value.ForStruct(new()
+                            {
+                                Fields =
+                                {
+                                    ["content"] = wkt::Value.ForString("How many moons does Mars have?")
+                                }
+                            }),
+                            ["output"] = wkt::Value.ForStruct(new()
+                            {
+                                Fields =
+                                {
+                                    ["content"] = wkt::Value.ForString("The planet Mars has two moons, Phobos and Deimos.")
+                                }
+                            })
+                        }
+                    })),
+                    ["messages"] = wkt::Value.ForList(wkt::Value.ForStruct(new()
+                    {
+                        Fields =
+                        {
+                            ["author"] = wkt::Value.ForString("user"),
+                            ["content"] = wkt::Value.ForString(prompt),
+                        }
+                    }))
+                }
+            }
+        };
+
+        var instances = new List<wkt::Value>
+        {
+            instance
+        };
+
+        // You can construct Protobuf from JSON.
+        var parametersJson = JsonConvert.SerializeObject(new
+        {
+            temperature = 0.3,
+            maxDecodeSteps = 200,
+            topP = 0.8,
+            topK = 40
+        });
+        // var parameters = wkt::Value.Parser.ParseJson(parametersJson);
+
+        // Or, you can construct Protobuf directly.
+        var parameters = new wkt::Value
+        {
+            StructValue = new wkt::Struct
+            {
+                Fields =
+                {
+                    { "temperature", new wkt::Value { NumberValue = 0.3 } },
+                    { "maxDecodeSteps", new wkt::Value { NumberValue = 200 } },
+                    { "topP", new wkt::Value { NumberValue = 0.8 } },
+                    { "topK", new wkt::Value { NumberValue = 40 } }
+                }
+            }
+        };
+
+        // Make the request.
+        var response = client.Predict(endpoint, instances, parameters);
+
+        // Parse the response and return the content.
+        var content = response.Predictions[0].StructValue.Fields["candidates"].ListValue.Values[0].StructValue.Fields["content"].StringValue;
```

Comment:
This should be simplified after the library is generated from new protos. May be a TODO here as well. So, that we don't miss out on it later.

Reply:
This is sample code, that users will see, we shouldn't leave TODOs for ourselves. Only TODOs, if any, might be those directed at developers  consuming the samples like this hypothetical example: "TODO: change this location to the location where you are storing your data.". And not even those are usual.

Also, we shouldn't comment on possible changes for which we don't have ETAs, etc.